### PR TITLE
FEAT: single video page

### PR DESCRIPTION
**Pull request by partha8**
# Note
There are a few features missing on the single video page, eg., like, add to playlist, etc. I will add them in the near future.

## FIles to check
1. [src/pages/Explore/components/VideoCard.js](https://github.com/partha8/weeb-culture/compare/video-player?expand=1#diff-78d0d7d55b48c8afef31793e90591f5c0460d8a82ad5b45a254d5564b37e288e)
2. [src/pages/VideoPlayback/VideoPlayback.js](https://github.com/partha8/weeb-culture/compare/video-player?expand=1#diff-3ee577ec3718fdbc3ddf866af968a6e046ce6d61a00b1f78c45a4f43b6e36141)
3. [src/utils/shuffleArray.js](https://github.com/partha8/weeb-culture/compare/video-player?expand=1#diff-2ad5bb2c6455723cd44ef620b7e6696cba86f76c15e58ecbcf2d8679197e840e)

### CODE REVIEW

**Review by OmKakatkar — APPROVED** [review]
Like:
* The use of barrel imports
* Folder structure with pages and utilities separated

**Review by Maianki — APPROVED** [review]
Small PR everything looks good. Nothing to suggest.
The UI looks superb man great work there.
Small suggestion - On the signup page if I click on the show password icon the confirm password show icon is also getting changed. Do fix it.

File: src/utils/shuffleArray.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Fisher-Yates (aka Knuth) Shuffle.
```
Comment (Vishnu-Aithal) [inline src/utils/shuffleArray.js:1]: suggestion: can use sort function to shuffle array
ex - 
const shuffledArr = arr.sort((a, b) => [1, -1][Math.floor(Math.random() * 2)]);

**Review by Vishnu-Aithal — APPROVED** [review]
Code is Very Declarative and Readable!
Good to merge !
only small suggestion

File: src/utils/shuffleArray.js (hunk 1 above)
Comment (partha8) [reply src/utils/shuffleArray.js:1]: Yes, I actually came across this too, but there are a few hiccups with this approach. I am linking a few articles which suggest fisher-yates algorithm is a better one.
1. https://javascript.info/task/shuffle
2. https://dev.to/codebubb/how-to-shuffle-an-array-in-javascript-2ikj

**Comment by partha8** [comment]
Thank you @Maianki , @OmKakatkar and @Vishnu-Aithal for the reviews!

File: src/utils/shuffleArray.js (hunk 1 above)
Comment (Vishnu-Aithal) [reply src/utils/shuffleArray.js:1]: Thanks for the articles . Went through it . Yep agree fisher-yates is the better one.

**Pull request merged**